src/driver/Firestore/InProcessFirestore.ts

```diff
@@ -700,6 +700,27 @@ interface IChildMeta {
 }
 
 export class InProcessFirestoreDocRef implements IFirestoreDocRef {
+    static validateNoUndefinedFields(
+        updateDelta: IFirestoreDocumentData,
+    ): void {
+        const undefinedFields = InProcessFirestoreDocRef.extractUndefinedFields(
+            updateDelta,
+        )
+        if (undefinedFields.length > 0) {
+            throw new Error(
```

Comment:
I believe there is a Firestore error - Might be able to use that here.
Can you ensure we throw the same code as Firestore / message